book/src/Example.md

```diff
@@ -0,0 +1,95 @@
+# Example
+
+The
+[HostPath](https://github.com/kubernetes-csi/drivers/tree/master/pkg/hostpath)
+can be used to provision local storage in a single node test. This
+section shows how to deploy and use that driver in Kubernetes.
+
+### Deployment
+
+The following .yaml file will deploy a `csi-pod` with the HostPath CSI driver and the
+[sidecar containers](CSI-Kubernetes.md#sidecar-containers) which integrate the driver into Kubernetes.
+Use it with a cluster that is [set up for CSI](Setup.html), either by copying it into a local file or
+by referencing the latest version directly:
+```
+kubectl create -f https://raw.githubusercontent.com/kubernetes-csi/docs/master/book/src/example/csi-setup.yaml
+```
+
+```yaml
+{{#include example/csi-setup.yaml}}
+```
+
+
+### Usage
+
+Dynamic provisioning was enabled by the .yaml file above via the
+`csi-hostpath-sc` storage class. We can use this to create and claim a
+new volume:
+```
+kubectl create -f https://raw.githubusercontent.com/kubernetes-csi/docs/master/book/src/example/csi-pvc.yaml
+```
+
```

Comment:
Do you think you could add the output of ```kubectl get pv```console? I think it will help a person who uses this guide to validate locally observed results.

Reply:
The output is already present, under "Troubleshooting". Is that too late because users won't read that far?

I can also interleave it with the commands here.

Reply:
I think adding the output command here will help a user to validate results he/she observing locally.  Troubleshooting section is great, but the user needs an indicator.  Comparing what you provide in the document with what he/she sees locally is good indicator and if something goes wrong then they can refer to the troubleshooting section.

Reply:
Okay, I'll make the usage section a bit more "interactive" by also including output.

Reply:
Done. I made the "Usage" section a mixture of commands+output, so one can see immediately what the result should. The "Troubleshooting" section now focuses on specific problems or aspects and does not show anymore what a working cluster looks like, because that already came earlier.

I also added something about VolumeAttachment.